temporal-sdk/src/main/java/io/temporal/client/WorkflowOptions.java

```diff
@@ -131,20 +131,24 @@ public Builder setWorkflowId(String workflowId) {
      * Specifies server behavior if a completed workflow with the same id exists. Note that under no
      * conditions Temporal allows two workflows with the same namespace and workflow id run
      * simultaneously.
-     * <li>
      *
-     *     <ul>
-     *       AllowDuplicateFailedOnly is a default value. It means that workflow can start if
-     *       previous run failed or was canceled or terminated.
-     * </ul>
+     * Default value if not set: AllowDuplicate
      *
-     * <ul>
-     *   AllowDuplicate allows new run independently of the previous run closure status.
-     * </ul>
+     * <li>
+     *  <ul>
+     *    AllowDuplicate allows new run independently of the previous run closure status. The previous run still needs to be closed or the new run will be rejected.
+     *  </ul>
+     *  <ul>
+     *    AllowDuplicateFailedOnly allows new run if previous run failed or was canceled or terminated.
+     *  </ul>
+     *  <ul>
+     *    RejectDuplicate doesn't allow new run independently of the previous run closure status.
+     *  </ul>
+     *  <ul>
+     *    TerminateIfRunning same as AllowDuplicate, but if there is an existing non-closed run in progress, it will be terminated.
```

Comment:
```suggestion
     *    TerminateIfRunning is the same as AllowDuplicate, but if there exists a not-closed run in progress, it will be terminated.
```